sdk/rust/src/transaction/execute.rs

```diff
@@ -206,6 +207,10 @@ where
     fn response_pre_check_status(response: &Self::GrpcResponse) -> crate::Result<i32> {
         Ok(response.node_transaction_precheck_code)
     }
+
+    fn validate_checksums_for_ledger_id(&self, ledger_id: &LedgerId) -> Result<(), Error> {
```

Comment:
transaction has a few other fields that may need checksum validation, like `node_account_ids`, `payer_account_id`, the transaction ID itself has an account ID